src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 17px;
+  scroll-behavior: smooth;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__content {
+    width: 166px;
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__title {
+    color: $main-color;
+    font-size: 12px;
+    font-weight: $medium-font-weigth;
+    line-height: 18px;
+    padding: 40px 0 0;
+    margin: 0;
+    transition: 0.5s ease-in-out;
+  }
+
+  &__code {
+    margin-top: 4px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+  }
+
+  &__reviews-number {
+    margin: 0;
+    color: $main-color;
+  }
+
+  &__price-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price {
+    font-size: 12px;
+    margin: 0;
+  }
+
+  &__price-number {
+    color: $main-color;
+    font-size: 16px;
+    font-weight: $accent-font-weigth;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 40px;
+    margin-top: 16px;
+    box-sizing: border-box;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    background: $accent-color;
+    color: #fff;
+    font-weight: $accent-font-weigth;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    transition: 0.5s ease-in-out;
+
+    &:hover {
+      border-color: $accent-color;
+      color: $accent-color;
+      background: transparent;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
please make your hover smooth, read [this](https://www.w3schools.com/howto/howto_css_transition_hover.asp) article